packages/graphback-codegen-resolvers/src/api/targetResolverContext.ts

```diff
@@ -169,19 +169,20 @@ function createRelations(inputContext: InputModelTypeContext[]) {
     t.fields.forEach((f: InputModelFieldContext) => {
       if (f.isType) {
         if (f.directives.OneToOne || !f.isArray) {
-          let columnName = `${t.name.toLowerCase()}Id`;
+          let columnName = `${f.name}Id`;
           if (f.directives.OneToOne) {
-            columnName = f.directives.OneToOne.field;
+            columnName = `${f.directives.OneToOne.field}Id`;
```

Comment:
Do we still support directives? 
Is database migration library recomending to mix both?

Reply:
For now the idea was to recommend to use both. As you said we cannot do all in one PR, There is a lot of changes required to replace relationship directives with annotations.